README.md

```diff
@@ -1,2 +1,122 @@
-# lambeq-docs
-User manual and example notebooks for the lambeq QNLP library
+# lambeq documentation
+
+[![lambeq logo](https://cqcl.github.io/lambeq-docs/_static/lambeq_logo.png)](//cqcl.github.io/lambeq-docs)
+
+![Build status](https://github.com/CQCL/lambeq-docs/actions/workflows/build_test.yml/badge.svg)
+[![License](https://img.shields.io/github/license/CQCL/lambeq-docs)](LICENSE)
+[![PyPI version](https://img.shields.io/pypi/v/lambeq)](//pypi.org/project/lambeq)
```

Comment:
Not sure if we keep Pypi buttons here for lambeq or remove them altogether. What Tket does?

Reply:
ok let's leave them then